src/backend/distributed/transaction/transaction_management.c

```diff
@@ -325,7 +327,18 @@ CoordinatedTransactionCallback(XactEvent event, void *arg)
 			/* handles both already prepared and open transactions */
 			if (CurrentCoordinatedTransactionState > COORD_TRANS_IDLE)
 			{
-				CoordinatedRemoteTransactionsAbort();
+				/*
+				 * Since CoordinateRemoteTransactionsAbort may cause an error and it is
+				 * not allowed to error out at that point, swallow the error if any.
```

Comment:
It'd be good to describe the error we found.

In particular, CreateWaitEventSet may throw an error when out of file descriptors.